postgrenosql/src/main/java/com/yahoo/ycsb/db/PostgreNoSQLDBClient.java

```diff
@@ -0,0 +1,293 @@
+/*
+  Copyright (c) 2012 - 2015 YCSB contributors. All rights reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you
+  may not use this file except in compliance with the License. You
+  may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+  implied. See the License for the specific language governing
+  permissions and limitations under the License. See accompanying
+  LICENSE file.
+ */
+package com.yahoo.ycsb.db;
```

Comment:
nit: it's nicer if the binding is in its own package, i.e. com.yahoo.ycsb.db.postgresnosql

(this is a nice-to-have and not something you need to alter for this PR to be accepted)